src/routes/terms-of-service/Content.svelte

```diff
@@ -0,0 +1,166 @@
+<script>
+  import { fly } from "svelte/transition";
+</script>
+
+<style>
+  h2 {
+    font-size: 38px;
+  }
+
+  h3 {
+    font-size: 20px;
+    margin-bottom: 2em;
+    margin-top: 2em;
+  }
+
+  h4 {
+    font-size: 16px;
+    margin-bottom: 0px;
+    margin-top: 2em;
+  }
+  h2,
+  h3,
+  h4 {
+    color: var(--color-ink-50);
+  }
+
+  h3,
+  h4 {
+    font-family: "Metropolis";
+  }
+
+  p,
+  ul {
+    color: var(--color-marketing-gray-70);
+  }
+
+  hr {
+    border-top: 2px dotted var(--color-marketing-gray-50);
+    margin-bottom: 2rem;
+  }
+</style>
+
+<div in:fly={{ duration: 800, y: 5 }}>
+  <h2>Our Privacy Consent Notice</h2>
+
+  <p>
+    Some sort of introduction lorem ipsum dolor sit amet, consectetur adipiscing
+    elit, sed do eiusmod tempor incididunt ut.
+  </p>
+
+  <hr />
+
+  <p>
+    Pioneer is an experimental initiative led by Mozilla to better understand
```

Comment:
Should we just `s/Pioneer/Ion` on this page?

Reply:
Yes, this copy will change in the very near future anyway.